subsys/bluetooth/audio/tbs.c

```diff
@@ -1026,6 +1030,8 @@ static uint8_t retrieve_call(struct tbs_service_inst *inst,
 
 	hold_other_calls(inst, 1, &ccp->call_index);
 
+	notify_calls(inst);
```

Comment:
The same applies here. Adding `notify_calls` may sent double notifications.

Reply:
Ok, I have done the changes. Thanks for your comment.